client/src/pages/Application.js

```diff
@@ -104,7 +106,26 @@ function Application() {
     }, //  Step 2
     {
       intro: <h1>step 3 intro here</h1>,
-      content: <h1>step 3 content here</h1>,
+      content: (
+        <div>
+          <Meetings
+            title="Interview Information"
+            status={
+              <StatusUpdate
+                title="Interview Info"
+                ambassador="Dhanush"
+                phone="123-456-7890"
+                email="test@tap.com"
+                date="1/1/2022"
+                time="6-7:00PM"
+                location="Zoom"
+              />
+            }
+            // textCard={<div>HELLO THERE</div>}
+            interviewInfo={<InterviewInfo contingent title="After Interviews" />}
+          />
+        </div>
```

Comment:
Can you get rid of the `div` wrapping the `Meetings` component here so that the card can expand to take up the full width of the page?